gradle.properties

```diff
@@ -14,3 +14,4 @@ org.gradle.jvmargs=-Xmx4g \
 --add-opens jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED
 # Could be moved to sonar properties after https://sonarsource.atlassian.net/browse/SONARGRADL-134
 systemProp.sonar.gradle.skipCompile=true
+version=24.3.1-SNAPSHOT
```

Comment:
we removed version from gradle.properties intentionally.  Is this an errant add?

Reply:
I had to add it to be able to use the Jenkins build from source. Sally found that problem in the scripts and I think it should get fixed soon. Once that is done I can get rid of it.

Reply:
removed